src/main/java/uk/gov/hmcts/reform/sscs/docmosis/config/PdfDocumentConfig.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.sscs.docmosis.config;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Data
+@Component
```

Comment:
This will mean we create one of these if the class is on our class path don't think we wanted that?

Reply:
We create all the PDF object just the Docmosis ones we don't